app/assets/javascripts/discourse/app/components/topic-timer-info.js

```diff
@@ -130,11 +132,10 @@ export default Component.extend({
     if (statusType === "silent_close") {
       statusType = "close";
     }
-
-    if (this.basedOnLastPost) {
-      return `topic.status_update_notice.auto_${statusType}_based_on_last_post`;
-    } else {
-      return `topic.status_update_notice.auto_${statusType}`;
+    if (this.basedOnLastPost && statusType === "close") {
```

Comment:
`close_after_last_post` is not a "real" type in the DB, it is just the `based_on_last_post` boolean combined with the `close` timer type.